### PR TITLE
allow set custom clusterIP

### PR DESCRIPTION
**Pull request by JoniJnm**
I would like to have this option :D

### CODE REVIEW

**Review by desaintmartin — CHANGES_REQUESTED** [review]
thanks for the contribution! Could you bumb minor version of Chart.yaml?

**Comment by JoniJnm** [comment]
directly in Chart.yaml? done, version set to 5.4.1

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.4.0
+version: 5.4.1
```
Comment (desaintmartin) [inline aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml:17]:
```suggestion
version: 5.5.0
```
According to semver, this is new feature.

**Review by desaintmartin — CHANGES_REQUESTED** [review]

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml (hunk 1 above)
Comment (JoniJnm) [reply aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml]: yes, sorry. Done

**Review by desaintmartin — APPROVED** [review]

**Pull request merged**